velox/functions/FunctionRegistry.h

```diff
@@ -76,4 +76,8 @@ resolveVectorFunctionWithMetadata(
 /// Clears the function registry.
 void clearFunctionRegistry();
 
+/// Get the function metadata corresponding to functionName.
+exec::VectorFunctionMetadata getFunctionMetadata(
```

Comment:
Maybe we should add a test to retrieve one or two function metadata and check? At least get the code exercised in this PR?
Or is that the plan in subsequent PRs?

Reply:
Thanks for the feedback, yes it would be better to test this path in Velox but since this function is only called in the Prestissimo PR, the unit tests were added in `presto-native-execution` instead: https://github.com/prestodb/presto/pull/22332/files#diff-714d30531f66c936686bcfdcb981aa7b8d598c1801f08057380f81eaa0c35983 . Could you please confirm if I should add unit tests in this PR too?

Reply:
@pramodsatya : Yes, lets definitely add a test here.

Reply:
Added unit tests for this function, thanks.